review input and rover photo (#2141)
OpenLiberty/blogs: posts/2022-03-31-nexus-conf.adoc

author_picture: https://avatars3.githubusercontent.com/GraceJansen
author_github: https://github.com/GraceJansen
seo-title: MicroProfile, Jakarta EE and Open Liberty at DEV NEXUS 2022 - OpenLiberty.io
seo-description: A helpful summary of the Devnexus 2022 workshops and talks on MicroProfile, Jakarta EE and Open Liberty.
blog_description: "A helpful summary of the Devnexus 2022 workshops and talks on MicroProfile, Jakarta EE and Open Liberty."

> open-graph-image: https://openliberty.io/img/twitter_card.jpg

Reviewer: ```suggestion
open-graph-image: https://openliberty.io/img/devnexus-2022-conf-logo.png
```

> https://devnexus.org/[DevNexus], the longest-running and largest Java Conference in the USA will be happening this year in Atlanta, CA, from April 12-14. As in-person conferences make a come back in 2022, this will be a great opportunity to re-connect with the Java community. 

Reviewer: ```suggestion
https://devnexus.org/[DevNexus], the longest-running and largest Java Conference in the USA will be happening this year in Atlanta, GA, from April 12-14. As in-person conferences make a comeback in 2022, this will be a great opportunity to re-connect with the Java community. 
```

> And of course, members of the Open Liberty team will be on-site and hosting multiple sessions and doing demonstrations and labs in the IBM booth. If you're also going to be there, deciding which of the great talks to see can be a bit of a challenge. So we've put together a helpful summary of the sessions on https://microprofile.io/[MicroProfile], https://jakarta.ee/[Jakarta EE] and https://openliberty.io/about/[Open Liberty].

Reviewer: ```suggestion
And of course, members of the Open Liberty team will be on-site hosting multiple sessions and running demonstrations and labs in the IBM booth. If you're also going to be there, deciding which of the great talks to see can be a bit of a challenge. So we've put together a helpful summary of the sessions on https://microprofile.io/[MicroProfile], https://jakarta.ee/[Jakarta EE] and https://openliberty.io/about/[Open Liberty].
```

=== Join us

> Key activities

Reviewer: ```suggestion
We'd love to see you at these key activities!
```

> 

Reviewer: I think this section might benefit from the definition list syntax, which gives you some special heading styles on your list items- see suggestions

Author: agree

> Jakarta EE Reception - Tuesday, May 12 @ 17:30 to 19:30 (room #41) - Join the Jakarta EE community for some great discussions on the ever changing landscape in enterprise Java and the future of cloud-native innovation that Jakarta EE represents, while enjoying drinks and snacks on IBM.

Reviewer: ```suggestion
Jakarta EE Reception - Tuesday, May 12 @ 17:30 to 19:30 (room #41):: 
Join the Jakarta EE community for some great discussions on the ever-changing landscape in enterprise Java and the future of cloud-native innovation that Jakarta EE represents, while enjoying drinks and snacks on IBM.
```

> Book Signing - Wednesday, May 13 @ 15:20 (IBM Booth) - Join IBMer, Java Champion, and co-author of a new eBook, “Practical cloud-native Java development with MicroProfile”, https://twitter.com/emilyfhjiang[Emily Jiang] as she signs and gives away copies of this popular publication.

Reviewer: ```suggestion
Book Signing - Wednesday, May 13 @ 15:20 (IBM Booth)::
Join IBMer, Java Champion, and co-author of a new eBook, “Practical cloud-native Java development with MicroProfile”, https://twitter.com/emilyfhjiang[Emily Jiang] as she signs and gives away copies of this popular publication.
```
